smartseq2_single_sample/check_reference/reference_check_wf.wdl

```diff
@@ -0,0 +1,43 @@
+import "star.wdl" as RunStar
+import "featurecounts.wdl" as fc
+
+workflow CheckReference {
+
+  File input_metadata
+  Array[Object] metadata = read_objects(input_metadata)
+  String sra_dir = "gs://broad-dsde-mint-dev-teststorage/patel_ap/"
+
+  scatter(record in metadata) {
+    call RunStar.Star as Star {  
+      input:
+        input_fastq_read1 = sra_dir+record.sraID+"_1.fastq.gz",
+        input_fastq_read2 = sra_dir+record.sraID+"_2.fastq.gz",
+        gtf = record.gtf,
+        star_genome = record.star_genome,
+        sample_tag = record.sraID+"_"+record.length+"_"+record.ref_name,
+        pu_tag = record.sraID+"_"+record.length+"_"+record.ref_name,
+        lib_tag = record.sraID+"_"+record.length+"_"+record.ref_name,
+        id_tag = record.sraID+"_"+record.length+"_"+record.ref_name
+    }
+
+    call fc.FeatureCountsUniqueMapping as uniqcount {
+      input:
+        aligned_bam = Star.output_bam,
+        gtf = record.gtf,
+        fc_out = record.sraID+"_"+record.length+"_"+record.ref_name
+    }
+
+    call fc.FeatureCountsMultiMapping as multcount {
+      input:
+        aligned_bam=Star.output_bam,
+        gtf = record.gtf,
+        fc_out = record.sraID+"_"+record.length+"_"+record.ref_name
+    }
+ }
+
+ output {
+   Star.*
```

Comment:
I think if using .* I would maybe comment what outputs you're using here; most people will go to the workflow wdl first, and since we have our tasks in individual files, they would then have to open up each individual file to figure out what is output here.